package/kedro_viz/server.py

```diff
@@ -398,11 +404,16 @@ def format_pipeline_data(
                 "full_name": getattr(node, "_func_name", str(node)),
                 "tags": sorted(node.tags),
                 "pipelines": [pipeline_key],
+                "modular_pipelines": [],
```

Comment:
This has to be a `set` but `set` can't be JSON serialisable so I did a quick list here to demo